.ci/publish-website

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Datawire. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+set -o errexit
+set -o nounset
+set -o verbose
+
+remote_url=https://d6e-automaton:${GH_TOKEN}@github.com/datawire/getambassador.io.git
+
+rm -rf /tmp/getambassador.io
+git clone -b master "$remote_url" /tmp/getambassador.io
+cd /tmp/getambassador.io
+npm run pull-docs
```

Comment:
This is where the docs are pushed to getambassador.io and a new website build is triggered post the push